src/lib/pubkey/dilithium/dilithium_common/dilithium_algos.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Crystals Dilithium Internal Algorithms
+ *
+ * (C) 2021-2024 Jack Lloyd
+ * (C) 2021-2022 Manuel Glaser and Michael Boric, Rohde & Schwarz Cybersecurity
+ * (C) 2021-2022 René Meusel and Hannes Rantzsch, neXenio GmbH
+ * (C) 2024 René Meusel, Rohde & Schwarz Cybersecurity
+ *
+ * Botan is released under the Simplified BSD License (see license.txt)
+ */
+
+#ifndef BOTAN_DILITHIUM_ALGOS_H_
+#define BOTAN_DILITHIUM_ALGOS_H_
+
+#include <botan/internal/dilithium_types.h>
+
+namespace Botan {
+
+DilithiumPolyMatNTT dilithium_expand_A(StrongSpan<const DilithiumSeedRho> rho, const DilithiumConstants& mode);
```

Comment:
Nit/bikeshed: I think these should all be in a sub-namespace eg `Dilithium_F` and then without the `dilithium_` prefix

Reply:
Done.